src/pages/index.fr-fr.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import Layout from "../layouts";
+import { graphql, Link } from 'gatsby'
+import { BlogLatest, BlogRoll, IndexHelmet, MainTitle, Section, Videos } from "../common/Index";
+import "../stylesheets/index.scss";
+
+const tracingText = `
+La faculté d'attacher des programmes eBPF sur des « trace points » et des
+« probe points », que ce soit dans le noyau ou dans les applications
+utilisateurs, offre des possibilités inégalées en termes de visibilité. Les
+deux vues produites par l'introspection du noyau, d'une part, et du
+comportement des applications en cours, d'autre part, peuvent être combinées
+pour obtenir une perspective unique et déterminante sur le système, et pour
+résoudre les problèmes de performance. Des structures de données élaborées
+permettent de collecter des indicateurs statistiques et d'extraire
+l'information utile de manière efficace, sans passer par un export constant
+d'échantillons de données comme le font les systèmes traditionnels.
+`;
+
+const securityText = `
+eBPF permet d'observer et de comprendre tous les appels système, et de se
+placer au niveau des paquets et des sockets pour obtenir une vision de toutes
+les opérations réseau. La combinaison de ces informations forme une fondation
+extraordinaire pour construire de nouvelles approches pour sécuriser les
+systèmes d'information. Alors même que le filtrage des appels système, des flux
+réseaux, et le suivi du contexte d'exécution des processus sont typiquement
+gérés par des solutions distinctes, eBPF offre à la fois vision et contrôle sur
+tous les composants, ouvrant ainsi la voie à un système de sécurisation qui
+porte sur un domaine plus large, et propose un contrôle plus poussé.
+`;
+
+const networkingText = `
+La programmabilité et la performance d'eBPF en font une solution idéale pour le
+traitement de paquets pour tous types d'applications réseau. Programmer avec
+eBPF permet d'analyser et d'implémenter de nouveaux protocoles, et d'appliquer
+une logique de routage des paquets pour répondre à des besoins en évolution
+constante, sans jamais quitter le contexte du noyau Linux. Le compilateur JIT
+(« Just-In-Time ») permet d'atteindre, à l'exécution des programmes, des
```

Comment:
Do we need a translation note here ("compilation à la volée")?

Reply:
Good idea, will add.

Reply:
It might even help harmonise the height of the two columns, as “monitoring” is currently longer :smile: